thrift/shared.thrift

```diff
@@ -1713,4 +1713,100 @@ struct RefreshWorkflowTasksRequest {
 
 struct FeatureFlags {
 	10: optional bool WorkflowExecutionAlreadyCompletedErrorEnabled
-}
\ No newline at end of file
+}
+
+enum CrossClusterTaskType {
+  StartChildExecution
+  CancelExecution
+  SignalExecution
+}
+
+enum CrossClusterTaskFailedCause {
+  DOMAIN_NOT_ACTIVE
+  WORKFLOW_ALREADY_RUNNING
+  WORKFLOW_NOT_EXISTS
+}
+
+struct CrossClusterTaskInfo {
+  10: optional string domainID
+  20: optional string workflowID
+  30: optional string runID
+  40: optional CrossClusterTaskType taskType
+  50: optional i16 taskState
+  60: optional i64 (js.type = "Long") taskID
+  70: optional i64 (js.type = "Long") version
+  80: optional i64 (js.type = "Long") visibilityTimestamp
+}
+
+struct CrossClusterStartChildExecutionRequestAttributes {
+  10: optional string targetDomainID
+  20: optional string requestID
+  30: optional i64 (js.type = "Long") initiatedEventID
+  40: optional StartChildWorkflowExecutionInitiatedEventAttributes initiatedEventAttributes
+}
+
+struct CrossClusterStartChildExecutionResponseAttributes {
+  10: optional string runID
+}
+
+struct CrossClusterCancelExecutionRequestAttributes {
+  10: optional string targetDomainID
+  20: optional string targetWorkflowID
+  30: optional string targetRunID
+  40: optional string requestID
+  50: optional i64 (js.type = "Long") initiatedEventID
+  60: optional bool childWorkflowOnly
+}
+
+struct CrossClusterCancelExecutionResponseAttributes { 
+}
```

Comment:
Define empty struct here in case we want to return some information from the target cluster in the future. Same for `CrossClusterSignalExecutionResponseAttributes`